.eslintrc.cjs

```diff
@@ -0,0 +1,87 @@
+/* eslint-env node */
+
+module.exports = {
+  root: true,
+
+  env: {
+    browser: true,
+    es2020: true,
+    jest: true,
+  },
+
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 2020, // keep in sync with tsconfig.json
+    sourceType: "module",
+  },
+
+  // eslint-disable-next-line prettier/prettier
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:react/recommended",
+    "prettier",
+  ],
+
+  // eslint-disable-next-line prettier/prettier
+  plugins: [
+    "prettier",
+    "unused-imports", // or eslint-plugin-import?
+    "@typescript-eslint",
+    "react",
+    "react-hooks",
+  ],
+
+  // NOTE: Tweak the rules as needed when bulk fixes get merged
+  rules: {
+    // TODO: set to "error" when prettier v2 to v3 style changes are fixed
+    "prettier/prettier": ["warn"],
+
+    // TODO: set to "error" when all resolved, but keep the `argsIgnorePattern`
+    "@typescript-eslint/no-unused-vars": ["warn", { argsIgnorePattern: "^_" }],
+
+    // TODO: each one of these can be removed or set to "error" when they're all resolved
+    "unused-imports/no-unused-imports": ["warn"],
+    "@typescript-eslint/ban-types": "warn",
+    "@typescript-eslint/no-explicit-any": "warn",
+    "react/jsx-key": "warn",
+    "react-hooks/rules-of-hooks": "warn",
```

Comment:
IMO the rules-of-hooks rule should be `"error"` since conditionally-called hooks can cause really strange broken behaviors.

Reply:
Sounds perfect to me.